modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -0,0 +1,100 @@
+// Used in cicd/builds/running-entitled-builds.adoc
+
+[id="builds-running-entitled-builds-with-sharedsecret-objects_{context}"]
+= Running entitled builds with sharedsecret objects
+
+[role="_abstract"]
+You can run entitled builds with `SharedSecret` objects that import {op-system-base} Simple Content Access (SCA) certificates.  By using this feature, you can install entitled RPM packages during builds without copying your {op-system-base} subscription credentials and certificates into the builds' namespaces.
+
+This procedure relies on the newly-introduced OpenShift Shared Resources feature and the Insights Operator.
+
+[IMPORTANT]
+====
+This enhancement is only available if you enable the `TechPreviewNoUpgrade` feature set. These Technology Preview features are not part of the default features. Enabling this feature set cannot be undone and prevents upgrades. This feature set is not recommended on production clusters. See xref:../post_installation_configuration/cluster-tasks.adoc#post-install-tp-tasks[Enabling Technology Preview features using FeatureGates].
+====
+
+.Prerequisites
+
+* You must xref:../post_installation_configuration/cluster-tasks.adoc#post-install-tp-tasks[enable Technology Preview features using FeatureGates].
+
+.Procedure
+
+Under the [examples directory](examples) there is both a simple example of using a `SharedConfigMap` in a `Pod`, and an example of performing an OpenShift `Build` from a `BuildConfig` that uses Red Hat Entitlements available from your Red Hat Subscription when your subscription credentials are stored on your cluster via OpenShift Insights Operator, and resulting `Secret` for those credentials is made available to additional `Namespaces` via a `SharedSecret`.  Instructions for the `Pod` example are [here](docs/simple-example.md), and for the `BuildConfig` example are [here](docs/build-with-rhel-entitlements.md).
```

Comment:
Looks like you grabbed the upstream markdown doc as is.

Certainly that should be pruned to just cite the BuildConfig one, and instead of going through the instructions that depend on contents of that repo, you should grab the yaml from the files in that repo and cite the yaml here in the doc, like we do with other examples in the build docs.

i.e. https://docs.openshift.com/container-platform/4.9/cicd/builds/running-entitled-builds.html

Now, as you have the WIP label on this PR, perhaps you have such changes in progress :-)

But it not, and you need further clarification / guidance, let's get on a video conf and discuss.

Reply:
I replaced this with the content from the gdoc for the blog post.